conf/drupal/config/block.block.ajaxpattern.yml

```diff
@@ -1,6 +1,6 @@
 uuid: 3bfa4546-fcbf-4b3a-b468-cbcd60172bec
```

Comment:
@ddiestra Might be worth instead saving this for a cleanup ticket, but AFAICT we only use AjaxPatternBlock for these two alerts blocks, meaning we could probably safely remove this block, ajaxpattern_2, AjaxPatternBlock (see docroot/modules/custom/mayflower/src/Plugin/Block/AjaxPatternBlock.php), and its associated JS (docroot/modules/custom/mayflower/js/ajax-pattern-alerts.js)

Reply:
I don't see keeping this here impacting anything on rollout though, so I'm good leaving it as-is for now